app/views/projects/samples/attachments/_attachment.html.erb

```diff
@@ -1,19 +1,80 @@
-<tr
-  id="<%= dom_id(attachment) %>"
-  class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
->
+<tr id="<%= dom_id(attachment) %>" class="bg-white border-b dark:bg-slate-800 dark:border-slate-700">
+  <% if attachment.associated_attachment && attachment.metadata['direction'] == 'forward' %>
+  <th class="px-6 py-4">
```

Comment:
**nitpick:** These should be `td` not `th`, `th` should only be used within a `thead`

Reply:
Fixed in 65b34e490be98d1f85f675ebfd53b0c9c3d11019